system/jlib/jsocket.cpp

```diff
@@ -7001,3 +7001,5 @@ IWhiteListHandler *createWhiteListHandler(WhiteListPopulateFunction populateFunc
     return new CWhiteListHandler(populateFunc, roleFormatFunc);
 }
 
+static_assert(sizeof(IpAddress) == 16, "check size of IpAddress");
```

Comment:
Probably not true if we ever turn off IPV6 flag...

Reply:
I think the size is always 16bytes at the moment - even with ipv6 turned off.  If it changed we would want to know.  I will open a jira to remove the code that directly writes the memory rather than serializing it first.